plugins/modules/ec2_vpc_nat_gateway.py

```diff
@@ -153,6 +165,17 @@
     wait: yes
     wait_timeout: 300
     region: ap-southeast-2
+
+- name: Create new nat gateway using an allocation-id and tags.
+  community.aws.ec2_vpc_nat_gateway:
+    state: present
+    subnet_id: subnet-12345678
+    allocation_id: eipalloc-12345678
+    region: ap-southeast-2
+    tags:
+      - Tag1: tag1
```

Comment:
Does space after ":" is necessary?

Reply:
Interesting catch, that doesn't match with our ususal tag handling which would be

```suggestion
      Tag1: tag1
      Tag2: tag2
```

I think this is just a bug in the documentation here, the integration tests treat tags as a dict.

Reply:
@tremble I changed this in the documentation. Sorry for that.